docs/concepts/service-catalog/index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Service Catalog
+approvers:
+- chenopis
+---
+
+{% capture overview %}
+{% glossary_definition term_id="service-catalog" length="all" %}  
+
+A *Service Broker*, as defined by the [Open Service Broker API spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md), is an endpoint for a set of Managed Services offered and maintained by a third-party, which could be a cloud provider such as AWS, GCP, or Azure.
+Some examples of *Managed Services* are Microsoft Azure Cloud Queue, Amazon Simple Queue Service, and Google Cloud Pub/Sub, but they can be any software offering that can be used by an application, typically available via HTTP REST endpoints.
+
+{% endcapture %}
+
+
+{% capture body %}
+## Example use case
+
+An {% glossary_tooltip text="Application Developer" term_id="application-developer" %} wants to use message queuing as part of their application running in a Kubernetes cluster.
+However, they do not want to deal with the overhead of setting such a service up and administering it themselves.
+Fortunately, there is a cloud provider that offers message queuing as a *Managed Service* through their *Service Broker*.
+
+Using Service Catalog, the {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} can browse the list of Managed Services offered by a Service Broker, provision an instance of the message queuing service, and bind with it to make it available to the application within the Kubernetes cluster.
```

Comment:
This paragraph is also general information and should go above the example use case.

Reply:
Ok, I refactored that section to move that info into the summary.